Adapter.C

```diff
@@ -23,6 +23,10 @@ bool preciceAdapter::Adapter::configFileRead()
     // See also comment in preciceAdapter::Adapter::configure().
     try
     {
+#ifdef ADAPTER_TIMING_MODE
```

Comment:
The `ifdef`s all over the place are actually a bit distracting. I'm not sure whether it's worth the saving in timing(?). Maybe we can find a less nosiy solution (e.g. using macros) or just add the `ifdef`s for the final cout.